python-package/lets_plot/plot/geom_extras.py

```diff
@@ -16,23 +15,39 @@ def arrow(angle=None, length=None, ends=None, type=None):
 
     Parameters
     ----------
-    angle : numeric
-        The angle of the arrow head in degrees (smaller numbers produce narrower, pointier arrows). 
+    angle : int
+        The angle of the arrow head in degrees (smaller numbers produce narrower, pointer arrows). 
         Essentially describes the width of the arrow head.
-    length : numeric
+    length : int
         The length of the arrow head (px).
-    ends : ['last' \ 'first' | 'both']
-        Indicating which ends of the line to draw arrow heads.
-    type : ['open' | 'closed']
-        Indicating whether the arrow head should be a closed triangle.
+    ends : str
+        Indicating which ends of the line to draw arrow heads ['last' | 'first' | 'both'].
```

Comment:
Why not something like this:

Indicating which ends of the line to draw arrow heads. Possible values: 'last', 'first', 'both'.

I think this should be closer to plain text.